config/development.json

```diff
@@ -200,7 +200,7 @@
 		"themes/atomic-homepage-replace": true,
 		"themes/block-theme-previews": true,
 		"themes/discovery-lits": false,
-		"themes/discovery-lots": false,
+		"themes/discovery-lots": true,
```

Comment:
Is this on purpose? :)

Reply:
Yes, I'm just going to leave it on for development. I'm tired of changing it or adding the flag.